views/js/front/mollie_iframe.js

```diff
@@ -165,7 +165,7 @@ $(document).ready(function () {
 
                 $mollieCardToken.val(token.token);
                 isResubmit = true;
-                $form.submit();
+                $form[0].submit();
             });
```

Comment:
does this work as well for other PS versions ?

Reply:
Yes, it works for 1.7.5.2 aswell. For 1.6 PS there is a different js file

Reply:
all good, approving!